dhis-2/olor=auto

```diff
@@ -0,0 +1,5 @@
+# Netscape HTTP Cookie File
```

Comment:
I think this file is a mistake.

Reply:
Thanks, I usually catch that but missed it this time. Not sure whey it keeps appearing in my source tree! :)